aws/alb.go

```diff
@@ -143,15 +131,19 @@ func createLoadBalancer(svc elbv2iface.ELBV2API, spec *createLoadBalancerSpec) (
 
 	newLoadBalancer := resp.LoadBalancers[0]
 	loadBalancerARN := aws.StringValue(newLoadBalancer.LoadBalancerArn)
+	// TODO: if any of the following fail, should we delete the just created LB?
 	targetGroupARN, err := createDefaultTargetGroup(svc, name, spec.vpcID, spec.healthCheck)
+	if err != nil {
+		return nil, err
+	}
+  
 	newHTTPSListener, err := createListener(svc, loadBalancerARN, targetGroupARN, httpsListenerPort, elbv2.ProtocolEnumHttps, spec.certificateARN)
 	if err != nil {
-		// TODO: delete just created LB?
 		return nil, err
 	}
-	newHTTPListener, err := createListener(svc, loadBalancerARN, targetGroupARN, httpListenerPort, elbv2.ProtocolEnumHttp, "")
+
+  newHTTPListener, err := createListener(svc, loadBalancerARN, targetGroupARN, httpListenerPort, elbv2.ProtocolEnumHttp, "")
```

Comment:
Yes, used the GH's web UI to fix the conflict. My local branch has that fixed. Thanks for spotting it.